### PR TITLE
JuliAstro GSoC: implement NOVAS in Julia

### PR DESCRIPTION
**Pull request by barrettp**
Adding NOVAS project

### CODE REVIEW

**Comment by bsipocz** [comment]
This is a JuliaAstro project, isn't it? Then it should go in a juliaastro directory.

**Comment by barrettp** [comment]
Yes, I suppose so. Then how do I get it included in the GSOC projects 2021
directory?  I'm a novice at this.

**Comment by bsipocz** [comment]
It's basically the same you did, just rename the novas directory to juliaastro.

File: _projects/2021/novas/NOVAS.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+name: NOVAS
+desc: Implement a Julia version of the U.S. Naval Observatory Vector
+Astrometry Software (NOVAS) library and vectorize it to improve performance.
```
Comment (dpshelio) [inline _projects/2021/novas/NOVAS.md:4]:
```suggestion
desc: Implement a Julia vectorized version of the NOVAS library
```

File: _projects/2021/novas/NOVAS.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+# Different technologies needed
+ - Julia
+ - linear algebra
+ - astronomy/astrometry
+ - JPL ephemeris library
+collaborating_projects:
+# suborganisation(s) to which this project belongs.
+ - juliaAstro
+---
+This is an awesome project idea for suborg.
```
Comment (dpshelio) [inline _projects/2021/novas/NOVAS.md:30]:
```suggestion
Implement a Julia version of the U.S. Naval Observatory Vector Astrometry 
Software (NOVAS) library and vectorize it to improve performance.
```

**Review by dpshelio — CHANGES_REQUESTED** [review]
Some other small changes to make the build happy.

**Comment by bsipocz** [comment]
@giordano - would you or someone else be a second mentor for this project? If yes, please add the github handle, and we can merge the PR.

File: _projects/2021/juliaastro/NOVAS.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+ - Familiarity with the Julia programming language.
+ - Familiarity with linear algebra/numerical methods.
+ - Familiarity with astronomy/astrometry.
+difficulty: medium
+issues:
+# Related issues (if any)  to this project.
+mentors:
+# First person in contact; mentors may change before project starts.
+# GitHub handles
+ - pebarrett
```
Comment (giordano) [inline _projects/2021/juliaastro/NOVAS.md:16]: Yes, I'm happy to be the second mentor
```suggestion
 - pebarrett
 - giordano
```

**Review by bsipocz — APPROVED** [review]

**Review by bsipocz — APPROVED** [review]

**Pull request merged**